### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Oleh-Rybachuk**

### CODE REVIEW

**Review by etojeDenys — CHANGES_REQUESTED** [review]
add links to your PR

**Comment by Oleh-Rybachuk** [comment]
https://oleh-rybachuk.github.io/react_list-of-goods-hooks/

File: src/App.tsx [hunk 1]
```diff
@@ -15,48 +16,133 @@ export const goodsFromServer = [
...
+      case SortType.ALPHABET:
+        return good1.localeCompare(good2);
+
+      case SortType.LENGTH:
+        return good1.length - good2.length;
+
+      default:
+        return 0;
+    }
+  });
```
Comment (IvanFesenko) [inline src/App.tsx:47]: No need to run sort function in case when sortType is NONE

File: src/App.tsx [hunk 2]
```diff
@@ -15,48 +16,133 @@ export const goodsFromServer = [
...
+}
+
 export const App: React.FC = () => {
+  const defaultReoderOptions = {
+    sortType: SortType.NONE,
+    isReversed: false,
+  };
+
+  const [reorderOptions, setReorderOptions] = useState(defaultReoderOptions);
+
```
Comment (IvanFesenko) [inline src/App.tsx:63]: move useState to 35 row

File: src/App.tsx [hunk 3]
```diff
@@ -15,48 +16,133 @@ export const goodsFromServer = [
...
+  }
+
+  return visibleGoods;
+}
+
 export const App: React.FC = () => {
+  const defaultReoderOptions = {
+    sortType: SortType.NONE,
+    isReversed: false,
+  };
```
Comment (IvanFesenko) [inline src/App.tsx:60]: Move outside of component

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
Good job. Lets make some improvements

**Pull request closed without merging**